frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -686,6 +688,7 @@
         SSO_LOGIN_ENABLED = Boolean.parseBoolean(props.getProperty("sso.login.enabled", "false"));
         IS_COMPONENT_VISIBILITY_RESTRICTION_ENABLED = Boolean.parseBoolean(
             System.getProperty("RunComponentVisibilityRestrictionTest", props.getProperty("component.visibility.restriction.enabled", "false")));
+        DISABLE_CLEARING_REQUEST_FOR_PROJECT_WITH_GROUPS = props.getProperty("org.eclipse.sw360.disable.clearing.request.for.project.group", "DEPT");
```

Comment:
Is it required to add a default Department - `Dept` as disabled?

Although it is a rare chance that there will be a Department - `Dept`. 

if it is absolutely necessary , then I am fine with it.